src/cargo/ops/cargo_update.rs

```diff
@@ -554,6 +777,92 @@ fn fill_with_deps<'a>(
     }
 }
 
+/// Upgrade an existing requirement to a new version.
+/// Copied from cargo-edit.
+fn upgrade_requirement(req: &str, version: &semver::Version) -> CargoResult<Option<String>> {
```

Comment:
We should probably pull in the unit tests as well: https://github.com/killercup/cargo-edit/blob/d395bc51bf1fd7b97edeabadbb1df78d83643f03/src/version.rs#L295

Reply:
Fixed.